CMake/Findcereal.cmake

```diff
@@ -0,0 +1,56 @@
+#Findcereal.cmake
```

Comment:
Thought of sadding this to https://gitlab.kitware.com/cmake/cmake/-/tree/master/Modules ? :smiley:

Reply:
I will take a look at this as well. Thanks!